### DISCUSSION TITLE
Get data on match outcomes and OS of players

### DISCUSSION THREAD

**Discussion by realJPIrwin** [beyond-all-reason/Beyond-All-Reason#4015] [General]
I'm a statistician. I'd like to evaluate the match balancing, and potentially build a better balancing model. The data I need seem to exist on a database attached to https://www.beyondallreason.info/replays 

I was hoping to get data for say all 2024 8v8 ranked games. Who won, OS of players, country of players, time of match in GMT, 'rank/chev' of players, party ID of players (if they are in a party) and name of map file. 

I was hoping someone on the team could build that dataset for me so I won't have to automate something to scrape it off the site. I have a few different hypothesis about the balance of games I'd like to investigate. I think if I can build a great model for predicting who wins the match, you could then use that model to improve the match making algorithm and get more matches closer to a 50/50 chance of winning for each team. That will make the game more fun.

**Comment by Brian-Catcow-B** [comment]
See: https://github.com/beyond-all-reason/data-processing
